python/ray/dashboard/metrics_exporter/api.py

```diff
@@ -15,20 +15,24 @@ def authentication_request(url, cluster_id) -> AuthResponse:
     auth_requeset = AuthRequest(cluster_id=cluster_id)
     response = requests.post(url, data=auth_requeset.json())
     response.raise_for_status()
-    return AuthResponse.parse_obj(response.json())
+    return AuthResponse.parse_obj(json.loads(response.json()))
```

Comment:
Something similar to pydantic parse_raw? https://pydantic-docs.helpmanual.io/usage/models/#model-properties

Reply:
What about `parse_json_string`?

Reply:
Will just keep it as is for now.